ush/load_modules_run_task.sh

```diff
@@ -299,15 +299,21 @@ Call to \"module use\" command failed."
     # Load the .local module file if available for the given task
     #
     modulefile_local="${task_name}.local"
-    if [ -f ${modules_dir}/${modulefile_local} ]; then
-      module load "${modulefile_local}" || print_err_msg_exit "\
+    if [ ${task_name} = "run_anal_gsi" ]; then
+      source "${modules_dir}/${modulefile_local}" || print_err_msg_exit "\
+Loading .local module file for the specified task (task_name) failed:
+  modulefile_local = \"${modulefile_local}\""
+    else
+      if [ -f ${modules_dir}/${modulefile_local} ]; then
+        module load "${modulefile_local}" || print_err_msg_exit "\
```

Comment:
This chunk should not need to be modified to source the run_anal_gsi module file. Please revert.

Reply:
I made a new "run_anal_gsi.local", but it only works when use "source". Using "module load" does not work.

Reply:
It should. It only has "module load" commands in it. Was that previous version, maybe?

Reply:
The old (dev2) use module load to load the run environment for each task:
#
#-----------------------------------------------------------------------
#
# Purge modules and load the module file for the specified task on the
# current machine.
#
#-----------------------------------------------------------------------
#
  print_info_msg "$VERBOSE" "
Loading modules for task \"${task_name}\" ..."

  module purge

  module use "${modules_dir}" || print_err_msg_exit "\
Call to \"module use\" command failed."

  #

Reply:
I spend lots of time but somehow module load cannot load the right environment. The module load does load python right for model to use Jinja template.